packages/donations-app/app/components/checkout/checkoutController.js

```diff
@@ -226,10 +228,11 @@ angular.module('aliceApp')
                 postDonationConfirmation();
               } else {
                 if (result.data.secureModeNeeded) {
```

Comment:
We should check on the back-end side if there is no "secureModeNeeded" returned from mango pay (we don't have 100% guarantee over it) and raise an error. It could be communicated to user as an error.